### ISSUE TITLE
Provide link to GlotPress from WP-Admin

### ISSUE THREAD

**Issue by garretthyder** [GlotPress/GlotPress-WP#897]
Hello,

As a first time user it was hard to find the actual GlotPress instance, only after reading the installation instructions did I realize there's no link and I have to navigate directly.

It would be nice to provide a link in the plugins list like many other plugins do linkign to their settigns or tools pages. Or might be nice to provide an onboarding alert with the link or even have it in the wpadminbar.

Thank you

**Comment by ocean90** [comment]
Duplicate of #259.

**Comment by garretthyder** [comment]
Thanks @ocean90 srry I overlooked that. Subscribed to it

**Comment by toolstack** [comment]
And there's a [plugin for that](https://wordpress.org/plugins/gp-additional-links/).

**Comment by garretthyder** [comment]
Thx @toolstack got it hooked up here
